src/overmap_ui.cpp

```diff
@@ -803,10 +803,12 @@ tripoint display( const tripoint &orig, const draw_data_t &data = draw_data_t()
                 color_notes += string_format( "%s:%s, ", color_pair.first.c_str(),
                                               _( color_pair.second.c_str() ) );
             }
-            color_notes = color_notes.replace( color_notes.end() - 2, color_notes.end(), "." );
+            color_notes = color_notes.replace( color_notes.end() - 2, color_notes.end(), 
+                                               ".\r\nX:TEXT for custom symbol, Y;TEXT for color.\r\n" );
```

Comment:
It should be G; for color

Reply:
Is there a particular reason? G; maps to green, but if you want blue you have to type B;

I figured Y would be less confusing, especially since it comes after an X.

Reply:
Also, even though "Y" isn't listed, using it will simply produce yellow text - the default color for a note.

Reply:
I guess you're right here. One more thought though, would it better clarify the syntax if it is worded `SYMBOL:TEXT for custom symbol, COLOR;TEXT for color`?

Reply:
Actually, yeah. You have a point, so let's do away with all this ambiguity.

Reply:
![image](https://user-images.githubusercontent.com/6323641/45199854-36c7a380-b26e-11e8-9104-cfcd3118e911.png)

I added a couple of extra newlines in order to make the text more readable. I also couldn't keep `SYMBOL:TEXT for custom symbol, COLOR;TEXT for color` on the same line without breaking it almost at the end, so I split those in two separate lines. Does it look okay?